src/main/kotlin/pl/put/smartgarden/domain/device/Device.kt

```diff
@@ -1,88 +1,94 @@
 package pl.put.smartgarden.domain.device
 
+import org.hibernate.annotations.GenericGenerator
 import java.time.Instant
-import javax.persistence.CascadeType
-import javax.persistence.Column
-import javax.persistence.Entity
-import javax.persistence.Id
-import javax.persistence.JoinColumn
-import javax.persistence.OneToMany
-import javax.persistence.Table
+import javax.persistence.*
 
 @Entity
 @Table(name = "devices")
-data class Device(
```

Comment:
"Here we don’t use data classes with val properties because JPA is not designed to work with immutable classes or the methods generated automatically by data classes."
 https://spring.io/guides/tutorials/spring-boot-kotlin/#_persistence_with_jpa